cmd/controller-manager/app/controller-manager.go

```diff
@@ -249,6 +254,16 @@ func setInt(target *int, defaultValue int) {
 
 func setDefaultFederationConfig(fedConfig *corev1a1.FederationConfig) {
 	spec := &fedConfig.Spec
+
+	if len(spec.Scope) == 0 {
+		defaultValue := os.Getenv(defaultScopeEnv)
+		if len(defaultValue) != 0 {
+			if defaultValue != string(apiextv1b1.ClusterScoped) && defaultValue != string(apiextv1b1.NamespaceScoped) {
+				glog.Fatalf("DefaultScope must be Cluster or Namespaced; got %q", defaultValue)
```

Comment:
Please include the name of the env var in this message, e.g.

glog.Fatalf("%s must be Cluster or Namespaced; got %q", defaultScopeEnv, defaultValue)